frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1049,12 +1053,13 @@ export default ({
 
           this.latestEvents.push(serializedMessage)
 
-          if (this.ephemeral.scrolledDistance < 50) {
+          // When the current scroll position is nearly at the bottom and a new message is added, auto-scroll to the bottom.
+          if (this.ephemeral.scrollableDistance < 50) {
```

Comment:
`this.ephemeral.scrollableDistance < 50` here is an existing logic, it means the auto-scrolling to the new message is only executed when the current scroll position is near the bottom. My guess for the intention is that we prevent this behavior when the user is reading old messages in the chat, which feels like something we shouldn't remove/interrupt. Pls let me know if we need any adjustment to this though.